docs/source/support/compatibility.rst

```diff
@@ -43,6 +43,8 @@ The below lists with which Daml Connect version a new Ledger API version was int
 
    * - Ledger API Version
      - Daml Connect Version
+   * - 1.11
```

Comment:
Please also add an entry below in the list of changes.

Reply:
This change makes sens ? I was really not sure of what I was doing here.

Reply:
This change is fine. We introduce Ledger API version 1.11 in SDK 1.14. What is missing is an entry below that explains the changes in Ledger API version 1.11

Reply:
Does this change make sens ? I really not sure of what I am doing here.

Reply:
:+1: